tests/test_loss.py

```diff
@@ -152,3 +153,33 @@ def test_logsumexp():
     b = np.full(n, 10000, dtype='float64')
     desired = 10000.0 + np.log(n)
     assert_almost_equal(_logsumexp(b), desired)
+
+
+def test_baseline_least_squares():
+    rng = np.random.RandomState(0)
+
+    loss = _LOSSES['least_squares']()
+    y_train = rng.normal(size=100)
+    baseline_prediction = loss.get_baseline_prediction(y_train, 1)
+    assert baseline_prediction.shape == tuple()  # float
+    assert_almost_equal(baseline_prediction, y_train.mean())
+
+
+def test_baseline_binary_crossentropy():
+    loss = _LOSSES['binary_crossentropy']()
+    for y_train in (np.zeros(shape=100), np.ones(shape=100)):
+        y_train = y_train.astype(np.float32)
+        baseline_prediction = loss.get_baseline_prediction(y_train, 1)
+        assert baseline_prediction.shape == tuple()  # float
+        assert_all_finite(baseline_prediction)
+
+
+def test_baseline_categorical_crossentropy():
+    prediction_dim = 4
+    loss = _LOSSES['categorical_crossentropy']()
+    for y_train in (np.zeros(shape=100), np.ones(shape=100)):
+        y_train = y_train.astype(np.float32)
+        baseline_prediction = loss.get_baseline_prediction(y_train,
+                                                           prediction_dim)
+        assert baseline_prediction.shape == (1, prediction_dim)
+        assert_all_finite(baseline_prediction)
```

Comment:
For those loss test, one could check that applying `_inverse_link_function(base_line_prediction)` approximately recovers `y_train[0]`.